solutions/usaco-919.mdx

```diff
@@ -103,4 +101,36 @@ public class paintbarn {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+WIDTH = 1000
+
+barn = [[0 for _ in range(WIDTH + 1)] for _ in range(WIDTH + 1)]
+with open('paintbarn.in') as read:
+	rect_num, paint_req = [int(i) for i in read.readline().split()]
```

Comment:
```suggestion
	rect_num, paint_req = map(int, read.readline().split())
```

Reply:
just asking, is there any particular reason? i can commit this, sure, but just asking

Reply:
i think it looks cleaner, but it's up to you if you want to commit it.